src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/client/ProfessionalApiClient.java

```diff
@@ -672,6 +677,36 @@ public void retrieveOrganisationDetailsByUnknownStatus(String status, String rol
         return response.body().as(Map.class);
     }
 
+    public void deleteOrganisation(String organisationId, String role, HttpStatus status) {

```

Comment:
is the parameter 'String role' required here?

Reply:
yes for debbug purpose its usefull.

Reply:
Remove role param if it has not used